docs/examples/task_control.md

```diff
@@ -0,0 +1,88 @@
+# MAVROS Example — Flight Task Control
+## Overview
```

Comment:
I prefer multiline headers for H1/H2 in markdown, i think they just looks better in plain text...

```markdown
H1
==

H2
---
```

Also better to add some whitespace between header and paragraph.